src/components/Buttons/EndCallButton/EndCallButton.tsx

```diff
@@ -20,10 +20,15 @@ const useStyles = makeStyles((theme: Theme) =>
 
 export default function EndCallButton(props: { className?: string }) {
   const classes = useStyles();
-  const { room } = useVideoContext();
+  const { room, updateBackgroundSettings } = useVideoContext();
+
+  const handleDisconnect = () => {
+    updateBackgroundSettings({ type: 'none', index: 0 });
```

Comment:
I don't think we want to update the settings to 'none' after disconnecting. We want to keep what the user had set right?